weeks/week07/week07.md

```diff
@@ -1,3 +1,38 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## General information:
```

Comment:
Remove this subtitle and keep the summary here.

Reply:
Done